### DISCUSSION TITLE
Watch a file

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#13478] [Other Feature Feedback, Questions, & Ideas]
GitHub is more than just the place where developers post their code. Over the last few years, it evolved also into a huge repository of content published through blogs and docs. For many, it's the go-to place to stay up-to-date on what's new.

While GitHub offers us a way to get notified of changes to repos, the notifications options are limited. You can get notified of new releases or pull requests, but you can't get notified when a particular file has changed.

**I would suggest an option to watch a specific file.**

A good use case for this is to get an update on the new Javascript proposals from the [tc39/proposals](https://github.com/tc39) repository. 
The [README.md](https://github.com/tc39/proposals/blob/main/README.md) file in that repository contains a summary of all of the proposals, and following this one file will keep you up to date when new proposals are being added and when existing proposals have there stage changed.

**Comment by beauxq** [comment]
What I would like seems related to this.

I would like notifications when any of the code in the repository changes.

The problem with the current notification options is that there is no way to do this without also receiving notifications that don't change any code in the repository (e.g. opening a new pull request, or closing a pull request without accepting it). I don't want those notifications if they don't change any code in the repository.
